crates/rust-analyzer/src/semantic_tokens.rs

```diff
@@ -40,7 +40,7 @@ macro_rules! define_semantic_token_types {
 define_semantic_token_types![
     (ATTRIBUTE, "attribute"),
     (BOOLEAN, "boolean"),
-    (BUILTIN_TYPE, "builtinType"),
+    (BUILTIN_TYPE, "type.defaultLibrary"),
```

Comment:
Wait, did they ship `nested.types` in LSP/VS Code? I thought that that's only planned? 

Or does it work by some coincidence, where VS Code uses textmate scope for this semantic token, without anything establishing the mapping explicitly?

Reply:
Hmmm. I didn't think about this, I wonder if I should really be adding an attribute "defaultLibrary" rather than sending both "type" and "defaultLibrary" in one token "type.defaultLibrary".

Reply:
Ah, yeah, indeed, it is standard token modifier: https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#semantic-token-classification

SO, I think when we lower BUITIN_TYPE to lsp, we should snap this builtin modifier on top (there's a chance that lsp_types even has it as a constant)